crates/sui-open-rpc/src/examples.rs

```diff
@@ -254,11 +254,11 @@ impl RpcExampleProvider {
     fn get_objects_owned_by_address(&mut self) -> Examples {
         let owner = SuiAddress::from(ObjectID::new(self.rng.gen()));
         let result = (0..4)
-            .map(|_| ObjectInfo {
+            .map(|_| SuiObjectInfo {
```

Comment:
This is the bug in question--should be `SuiObjectInfo` because that's what `get_objects_owned_by_address` actually returns.